File: 1848-sum-of-unique-elements/sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        int sum = 0;
        int[] res = new int[101];
        for(int i:nums){
            res[i]++;
        }
        for(int i:nums){
            if(res[i]==1){
                sum += i;
            }
        }
        return sum;
    }
}

File: 1848-sum-of-unique-elements/sum-of-unique-elements.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        freq = {}
        for i in nums:
            freq[i] = freq.get(i,0)+1
        s = 0
        for key,value in freq.items():
            if value == 1:
                s += key
        return s